net/safe-search/Makefile

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2018 Gregory L. Dietsche <Gregory.Dietsche@cuw.edu>
+# This is free software, licensed under the MIT License
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=safe-search
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Gregory L. Dietsche <Gregory.Dietsche@cuw.edu>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/safe-search
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=Safe Search
+	PKGARCH:=all
+endef
+
+define Package/safe-search/description
+This package prevents adult content from appearing in search results by
+configuring dnsmasq to force all devices on your network to use Google and
+Bing's Safe Search IP addresses. This is designed to be approperiate for most
+businesses and families. The default filtering rules do not interfere with
+normal web browsing.
+endef
+
+define Package/safe-search/conffiles
+/etc/config/safe-search
+endef
```

Comment:
I think maybe this conffiles definition can be left out.  If it's about sysupgrade keep files, `/etc/config/` dir will be taken care of as the default behaviour.

Reply:
Without that, opkg remove safe-search won't leave /etc/config/safe-search if it has been modified. That's why I have it there. I think either opkg would need a bug fix, or this code should stay. Happy to do whatever you think is best!

> root@testRouter:~# opkg remove safe-search
> Removing package safe-search from root...
> Not deleting modified conffile /etc/config/safe-search.

Reply:
Thanks for the info.

I think package managers should not tamper with user generated data by default.  I remembered `apt-get remove` will keep config files while removing binaries, `apt-get purge` is used to clean all traces.

Let's keep the conffiles definition for now just in case.